app/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 32
     buildToolsVersion "30.0.3"
```

Comment:
You changed the compile SDK version, but not the build tools version. This is not good practice and gives off warnings in Android Studio.

Reply:
OK, I will restore it to the previous state